vendor/bat-native-ads/src/bat/ads/internal/ads/ad_events/search_result_ads/search_result_ad_event_handler_unittest.cc

```diff
@@ -114,10 +115,10 @@ class BatAdsSearchResultAdEventHandlerTest : public EventHandlerObserver,
     UnitTestBase::TearDown();
   }
 
-  void FireEvent(const mojom::SearchResultAdInfoPtr& ad_mojom,
+  void FireEvent(mojom::SearchResultAdInfoPtr ad_mojom,
                  const mojom::SearchResultAdEventType event_type) {
     event_handler_->FireEvent(
```

Comment:
DCHECK for both ad_mojom and event_type

Reply:
I think these DCHECKs are unnecessary here, as we are just passing these parameters to EventHandler::FireEvent, and there is already DCHECK there.